Dockerfile

```diff
@@ -1,5 +1,5 @@
-ARG GOLANG_VERSION=1.22
-ARG ALPINE_VERSION=3.19
+ARG GOLANG_VERSION=1.24
```

Comment:
current release candidate is `1.23`. `1.24` will be released next february

Reply:
my bad